src/OrleansAzureUtils/Providers/AzureProviderErrorCode.cs

```diff
@@ -16,5 +16,26 @@ internal enum AzureProviderErrorCode
         AzureTableProvider_DeleteError              = AzureTableProviderBase + 8,
         AzureTableProvider_InitProvider             = AzureTableProviderBase + 9,
         AzureTableProvider_ParamConnectionString    = AzureTableProviderBase + 10,
+
+        AzureBlobProviderBase                       = ProvidersBase + 200,
```

Comment:
This conflicts with [ProviderErrorCode.ShardedStorageProviderBase](https://github.com/dotnet/orleans/blob/a64fa8953e764ac391ebd549d395ac66374b545f/src/OrleansProviders/ProviderErrorCode.cs#L7)
I do not understand why we have these constants spread out in different classes though (probably different projects, but that kind of defeats the benefit of it), but make sure they don't overlap.